client-sdk/ts-web/core/playground/src/startPlayground.mjs

```diff
@@ -2,7 +2,7 @@
 
 /** @param {import('./../..')} oasis */
 export async function startPlayground(oasis) {
-    const nic = new oasis.client.NodeInternal('http://localhost:42280');
+    const nic = new oasis.client.NodeInternal('http://127.0.0.1:42280');
```

Comment:
:information_source: we had been listening on specifically ipv4 127.0.0.1 in the envoy proxy config, this fixes it not to try ipv6 localhost. thanks for the fix

Reply:
I should make that as a separate commit

Reply:
https://github.com/oasisprotocol/oasis-sdk/pull/1364/commits/a86df5d7fedf51ab36736a9deed11b3d83001303